src/components/Permissions/OpenappButton.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import CircleButton from 'cozy-ui/transpiled/react/CircleButton'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import OpenappIcon from 'cozy-ui/transpiled/react/Icons/Openapp'
+import AppLinker, { generateWebLink } from 'cozy-ui/transpiled/react/AppLinker'
+import { useClient } from 'cozy-client'
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+
+export const OpenappButton = ({ type, matchingQueryResultData }) => {
```

Comment:
@vdnj this is a nit, don't make the change because I say it: 

`OpenAppButton` is a "displayed component". It deals with App/Konnecteur and Link. 

To me the props should not be named `matchingQueryResultData`. It doesn't mean anything when reading the component without the usage context. Having a props named `appOrKonnectorData` instead of `matchingQueryResultData` gives us a lot more information about the component.